Apps/WebClient/src/NewClientApp/src/stores/immunization.ts

```diff
@@ -25,18 +25,11 @@ const defaultImmunizationDatasetState: ImmunizationDatasetState = {
     recommendations: [],
 };
 
-function immunizationSort(a: ImmunizationEvent, b: ImmunizationEvent) {
-    const firstDate = new DateWrapper(a.dateOfImmunization);
-    const secondDate = new DateWrapper(b.dateOfImmunization);
-
-    if (firstDate.isAfter(secondDate)) {
-        return 1;
-    }
-    if (firstDate.isBefore(secondDate)) {
-        return -1;
-    }
-    return 0;
-}
+const immunizationSort = (a: ImmunizationEvent, b: ImmunizationEvent): number =>
+    DateWrapperSortUtility.descendingByString(
```

Comment:
```suggestion
    DateWrapperSortUtility.ascendingByString(
```
Note that `firstDate.isBefore(secondDate)` returns -1 in this function, even though the order of the if statements is reversed.